.pre-commit-config.yaml

```diff
@@ -3,6 +3,8 @@ fail_fast: true
 exclude: |
   (?x)^(
     crates/ruff/resources/.*|
+    crates/ruff/src/rules/.*/snapshots/.*|
+    crates/ruff/src/rules/pep8_naming/mod.rs| # Uses "BA" as acronym.
```

Comment:
Sorry, forgot to mention that earlier: This one specifically might fit better in _typos.toml